Formula/console_bridge.rb

```diff
@@ -0,0 +1,25 @@
+class ConsoleBridge < Formula
+  desc "ROS-independent package for logging"
```

Comment:
Could this be a little more descriptive? I have not idea what ROS is

Reply:
@scpeters @j-rivero Any suggestion for the description? :smile: